### DISCUSSION TITLE
After changing claims, user's claims are not updated

### DISCUSSION THREAD

**Discussion by locona** [gladly-team/next-firebase-auth#250] [Q&A]
**Describe the bug**
After changing claims, user's claims are not updated

**Version**
v0.13.1-alpha.3

**To Reproduce**
1. Create a server-side function to update claims.
2. Call the prepared function and call the following on the frontend
```
const { claims } = uesAuthUser();
```

3. Claims will not be updated.

**Expected behavior**

I want the frontend useAuthUser claims to be updated after calling ` setCustomUserClaims` on the server side (like firebase functions).

**Additional context**

**Comment by locona** [comment]
Maybe there is a problem here.

https://github.com/gladly-team/next-firebase-auth/blob/main/src/withAuthUser.js#L58-L61

**Comment by shreyas-jadhav** [comment]
You need to call `AuthUser.getIdToken(true)`

This will refresh the instance and fetch latest claims.

**Reply by arbejd** [reply]
This does not fetch the latest claims on my setup at least. 

I just get my token back in the promise and nothing happens on the claim side. Needs to log out and in again to get claims updated. Are you certain this works?

0.13.3

**Reply by arbejd** [reply]
For future reference i got claims to update client side by doing `AuthUser.firebaseUser.getIdToken(true)` instead. I have no idea what the difference is ^^

**Reply by kmjennison** [reply]
Opened an issue: https://github.com/gladly-team/next-firebase-auth/issues/286

**Comment by locona** [comment]
@shreyas-jadhav 

What I want to do is to change the claims of all members except myself.

```
AuthUser.getIdToken(true)
```

If I put the above in, won't I have to do it every time when rendering?
